content/influxdb/cloud-dedicated/admin/databases/_index.md

```diff
@@ -91,20 +103,30 @@ operating cost of your cluster.
 
 **Default maximum number of columns**: 250
 
-Time, fields, and tags are each represented by a column in a table.
+Each row must include a time column, with the remaining columns representing
+tags and fields.
+As a result, a table can have one time column and up to 249 field and tag columns.
+
+If you attempt to write to a table and exceed the column limit, the write
+request fails and InfluxDB returns an error.
+
+When creating or updating a database, you can configure the table column limit to be
+lower than the default or up to 1000, based on your requirements.
+After you update the column limit for a database, the limit applies to newly
+created tables; it doesn't override the column limit for existing tables.
+
 Increasing your column limit affects your {{% product-name omit=" Clustered" %}}
 cluster in the following ways:
 
 {{< expand-wrapper >}}
-{{% expand "May adversely affect query performance" %}}
-
-At query time, the InfluxDB query engine identifies what table contains the queried
-data and then evaluates each row in the table to match the conditions of the query.
-The more columns that are in each row, the longer it takes to evaluate each row.
-
-Through performance testing, InfluxData has identified 250 columns as the
-threshold where query performance may be affected
-(depending on the shape of and data types in your schema).
+{{% expand "May adversely affect system performance" %}}
+
+InfluxData identified 250 columns as the safe limit for maintaining system
+performance and stability.
+Exceeding this threshold can result in
```

Comment:
Do we want to mention that 250 is a soft limit and users can raise the column limit to 1000 (a hard limit), but we don't recommend it?